apps/evm/src/containers/MarketTable/Apy/BoostBadge/index.tsx

```diff
@@ -0,0 +1,76 @@
+import { Tooltip, type TooltipProps } from 'components';
+import { Link } from 'containers/Link';
+import { useTranslation } from 'libs/translations';
+import type { AssetDistribution } from 'types';
+import { cn, formatPercentageToReadableValue } from 'utilities';
+import { DistributionGroup } from './DistributionGroup';
+import starsIconSrc from './stars.svg';
+import type { Distribution } from './types';
+
+export interface BoostBadgeProps extends Omit<TooltipProps, 'children' | 'title'> {
+  assetDistributions: AssetDistribution[];
+}
+
+export const BoostBadge: React.FC<BoostBadgeProps> = ({
+  className,
+  assetDistributions,
+  ...otherProps
+}) => {
+  const { t, Trans } = useTranslation();
+
+  const tokenDistributions = assetDistributions.reduce<Distribution[]>((acc, d) => {
+    // Filter out 0% distributions
+    if (d.apyPercentage.isEqualTo(0)) {
+      return acc;
+    }
+
+    if (d.type === 'merkl') {
+      const distribution: Distribution = {
+        name: d.description || t('marketTable.apy.primeBadge.tooltip.defaultMerklRewardName'),
+        value: formatPercentageToReadableValue(d.apyPercentage),
+        description: (
+          <Trans
+            i18nKey="marketTable.apy.primeBadge.tooltip.externalRewardDescription"
+            components={{
+              AppLink: (
+                <Link
+                  target="_blank"
+                  href={d.claimUrl || 'https://merkl.angle.money/'}
```

Comment:
About the claimUrl, if the type is merkl there is always a claimUrl (could be either Ignite or Merkl currently). If we keep the fallback, I'd use https://app.merkl.xyz/

Reply:
I'd also think how this would look like if there were multiple distributions with the same URL, I'd think this would start looking repetitive

Reply:
Ok I see, good to know. No need for a fallback in this case.

The URL won't be visible, as it will be hidden under an "official app" text link. It is true that the same message would be shown multiple times if there were multiple Merkl rewards, but I don't think this is necessarily a bad thing as this way users will be used to it. An alternative would be to have a different UI for when there are multiple Merkl rewards, but to be honest I don't know how often this will be the case and so whether it is worth it (we'd also have to think about cases like the Ignite campaign where although it is a Merkl campaign, rewards are claimed through a separate app).